tools/rum-models-generator/Sources/CodeGeneration/Print/SwiftPrinter.swift

```diff
@@ -102,6 +106,9 @@ public class SwiftPrinter: BasePrinter, CodePrinter {
             }
 
             printComment(property.comment)
+            if let attribute = configuration.accessLevel.attribute as? String {
+                writeLine("\(attribute)")
+            }
```

Comment:
```suggestion
```
You don't need the attribute on properties and methods since it's set at the type level 👍